.github/workflows/backend-converted-types-check.yml

```diff
@@ -0,0 +1,46 @@
+name: Backend-Converted-Types-Check
+on:
+  push:
+    paths:
+      - 'backend/**'
+      - 'frontend/**'
```

Comment:
```suggestion
```
this might save `compute time` and could be unnecessary if I understood the `usecase` correctly

Reply:
someone could change the types in frontend-dir, but you are right its better to use `frontend/types/api/**`

Reply:
updated

Reply:
I guess you still missed this line and only updated `pull_requests` 🤔

Reply:
yep, fixed